oam/install_scripts/syslogSetup.sh

```diff
@@ -13,12 +13,9 @@ syslog_conf=nofile
 rsyslog7=0
 
 user=`whoami 2>/dev/null`
-group=user
 
-SUDO=" "
-if [ "$user" != "root" ]; then
-        SUDO="sudo "
-fi
+groupname=adm
+username=syslog
```

Comment:
having 2 vars named user and username is confusing.  also, groupname I'm not sure is ever assigned a group name.